### PR TITLE
lock foundry version

### PR DESCRIPTION
**Pull request by RensR**
Use `make foundry` to get the correct version

### CODE REVIEW

**Review by Fletch153 — DISMISSED** [review]

File: contracts/GNUmakefile [hunk 1]
```diff
@@ -1,7 +1,7 @@
 # ALL_FOUNDRY_PRODUCTS contains a list of all products that have a foundry
-# profile defined. Adding a product to this list will make it available for
-# snapshotting.
-ALL_FOUNDRY_PRODUCTS = vrf automation llo-feeds functions automation-dev shared
```
Comment (austinborn) [inline contracts/GNUmakefile:4]: `vrf` no longer using foundry?

**Review by austinborn — APPROVED** [review]

File: contracts/GNUmakefile [hunk 2]
```diff
@@ -37,9 +37,12 @@ abigen: ## Build & install abigen.
...
 	go install github.com/vektra/mockery/v2@v2.28.1
 
+.PHONY: foundry
+foundry: ## Install foundry.
+	foundryup --version nightly-5cfed63071ef1bb06f90190d2f678f546bfc971c
+
 .PHONY: foundry-refresh
-foundry-refresh:
-	foundryup
+foundry-refresh: foundry
```
Comment (RyanRHall) [inline contracts/GNUmakefile:45]: is this command even necessary anymore?

**Review by RyanRHall — APPROVED** [review]

File: contracts/GNUmakefile (hunk 2 above)
Comment (RensR) [reply contracts/GNUmakefile:45]: Yes, helped someone today. It doesn't just update Foundry, it fixes dependency issues when you switch between branches that have different Foundry dependencies. A must have for when we bump the std-lib the next time

Comment (RyanRHall) [reply contracts/GNUmakefile:44]: oh I just meant since `foundry-refresh` just calls `foundry` now - maybe we could avoid the duplication :)

**Review by RyanRHall — APPROVED** [review]

**Pull request merged**